### PR TITLE
pay by link implementation and example added

### PR DESCRIPTION
**Pull request by silviusomesan**

### CODE REVIEW

File: src/PayU/Alu/Response.php [hunk 1]
```diff
@@ -376,6 +388,7 @@ public function parseAdditionalParameters($parameters)
             'ECI',
             'CAVV',
             'TRANSID',
+            'URL_REDIRECT',
```
Comment (drealecs) [inline src/PayU/Alu/Response.php]: `parseAdditionalParameters` method is used for reading and parsing ALU response plugins. Let's keep it's scope to that and not add `URL_REDIRECT` here.

File: src/PayU/Alu/Response.php [hunk 2]
```diff
@@ -355,6 +348,25 @@ public function setRrn($rrn)
         $this->rrn = $rrn;
     }
 
+    /**
```
Comment (drealecs) [inline src/PayU/Alu/Response.php]: You made the DocBlock comments from above in one line.
While that looks a bit nicer, try to be consistent and have the new ones in the same way or not change the old ones.

File: examples/payByLink.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Include composer class autoloader
+ */
+require_once dirname(__FILE__) . '/../vendor/autoload.php';
+
+use PayU\Alu\Billing;
+use PayU\Alu\CardToken;
```
Comment (drealecs) [inline examples/payByLink.php]: unused imported class

File: examples/payByLink.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+ * Will throw different Exceptions on errors
+ */
+try {
+    /**
+     * Sends the Request to ALU and returns a Response
+     *
+     * See documentation for Response params
+     */
+    $response = $client->pay($request);
+    echo $response->getReturnMessage();
```
Comment (drealecs) [inline examples/payByLink.php]: please add some spacing... echoing string one immediately after another is not helping

File: src/PayU/Alu/Response.php (hunk 2 above)
Comment (silviusomesan) [inline src/PayU/Alu/Response.php]: My intention was to bring consistency in Response.php file.
I've changed the dockblock for all properties to be consistent across all project

File: src/PayU/Alu/Response.php (hunk 1 above)
Comment (silviusomesan) [inline src/PayU/Alu/Response.php]: fixed

File: examples/payByLink.php (hunk 3 above)
Comment (silviusomesan) [inline examples/payByLink.php]: fixed

File: examples/payByLink.php (hunk 4 above)
Comment (silviusomesan) [inline examples/payByLink.php]: changed the way how response is displayed in all examples as it is in basicExample.php (assuming that file was the reference)

File: src/PayU/Alu/Response.php (hunk 2 above)
Comment (drealecs) [inline src/PayU/Alu/Response.php]: OK, Looks good!

**Comment by drealecs** [comment]
Looks good. Is it ready to be merged?

**Comment by silviusomesan** [comment]
Yes, it can be merged. Bkm ALU is already in production. Thanks !

**Pull request merged**